tests/index.test.js

```diff
@@ -0,0 +1,21 @@
+const { configs, rules } = require('../index')
+const { checkExists } = require('../utils')
+
+describe('main export', () => {
+    it('has rules', () => {
+        checkExists(rules)
+        checkExists(rules['no-absolute-internal-imports'])
+        checkExists(rules['no-parent-imports'])
+    })
+
+    it('has configs', () => {
+        checkExists(configs)
+        checkExists(configs.recommended)
+        checkExists(configs.flow)
+        checkExists(configs.typescript)
+    })
+
+    it('has correct shapshot', () => {
+        expect(configs.recommended).toMatchSnapshot()
```

Comment:
will this not fail every time we change a recommended rule? What's the point?

Reply:
it is getting constructed - when we change something in the config we will know exactly what is changed. 

It is really simple to just pres `u`. This is an e2e check of the repo. I am also considering to add those tests for other rules.